.ci/pytorch/test.sh

```diff
@@ -491,6 +491,11 @@ test_perf_for_dashboard() {
         cp "$TEST_REPORTS_DIR/${backend}_with_cudagraphs_${suite}_${dtype}_${mode}_cuda_${target}.csv" \
           "$TEST_REPORTS_DIR/${backend}_cudagraphs_low_precision_${suite}_quant_${mode}_cuda_${target}.csv"
       fi
+      if [[ "$DASHBOARD_TAG" == *inductor_cpu-true* ]]; then
```

Comment:
Let's limited the inductor cpu performance nightly start from fp32 and amp inference firstly

Reply:
Done. I disabled training test. And only run float32 and amp inference for inductor cpu nightly